golang/simple_consumer.go

```diff
@@ -183,6 +184,19 @@ func (sc *defaultSimpleConsumer) wrapAckMessageRequest(messageView *MessageView)
 	}
 }
 
+func (sc *defaultSimpleConsumer) wrapForwardMessageToDeadLetterQueueRequest(messageView *MessageView) *v2.ForwardMessageToDeadLetterQueueRequest {
```

Comment:
There should not be the `ForwardMessageToDeadLetterQueueRequest` in SimpleConsumer.

Reply:
Is there any good way to customize the retry policy? For example, if there are two messages m and n in topic A, I want to retry 3 times for m, but 100 times for n, and the retry interval can be decided by the client